app/views/layouts/application.html.erb

```diff
@@ -6,16 +6,16 @@
     <%= csp_meta_tag %>
     <%= canonical_tag %>
     <%= tag :meta, property: "viewport", content: "width=device-width, initial-scale=1" %>
```

Comment:
Whilst you are here can you make this `name:` instead of `property:`. I was an error in the boilerplate.